js/api.js

```diff
@@ -0,0 +1,20 @@
+const URL = 'https://28.javascript.pages.academy/kekstagram';
+
+const load = async () =>
+  fetch(`${URL}/data`)
+    .then((response) => response.json())
```

Comment:
Лучше сразу проверить на `ok` ответ от сервера и если что-то не так, выбросить ошибку
```suggestion
   .then((response) => {
      if (response.ok) {
        return response.json();
      }
      throw new Error('Данные не загрузились');
    })
```